src/kudu/consensus/raft_consensus.cc

```diff
@@ -3464,6 +3464,33 @@ Status RaftConsensus::SetPendingConfigUnlocked(const RaftConfigPB& new_config) {
   return Status::OK();
 }
 
+Status
+RaftConsensus::ChangeTopologyConfig(const TopologyConfigPB &topology_config) {
```

Comment:
Should this be ChangeVoterDistribution for now, since other things in topology_config won't be processed.